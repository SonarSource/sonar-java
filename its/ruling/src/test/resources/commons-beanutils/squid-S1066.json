{
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/MethodUtils.java':[
1298,
],
}
