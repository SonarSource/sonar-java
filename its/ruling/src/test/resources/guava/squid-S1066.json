{
'com.google.guava:guava:src/com/google/common/cache/LocalCache.java':[
2673,
4322,
],
'com.google.guava:guava:src/com/google/common/collect/MapMakerInternalMap.java':[
3654,
],
'com.google.guava:guava:src/com/google/common/io/LineBuffer.java':[
54,
66,
],
}
