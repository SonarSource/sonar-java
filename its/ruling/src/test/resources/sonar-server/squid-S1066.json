{
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/source/HtmlTextDecorator.java':[
97,
],
}
