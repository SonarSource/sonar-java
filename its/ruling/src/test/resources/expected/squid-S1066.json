{
'project:com/sun/corba/se/impl/activation/ServerTool.java':[
449,
525,
],
'project:com/sun/corba/se/impl/corba/TypeCodeImpl.java':[
403,
667,
944,
949,
957,
962,
981,
],
'project:com/sun/corba/se/impl/encoding/BufferManagerReadStream.java':[
212,
],
'project:com/sun/corba/se/impl/encoding/CDRInputObject.java':[
214,
],
'project:com/sun/corba/se/impl/interceptors/CodecFactoryImpl.java':[
100,
],
'project:com/sun/corba/se/impl/interceptors/RequestInfoImpl.java':[
846,
862,
],
'project:com/sun/corba/se/impl/io/IIOPInputStream.java':[
279,
],
'project:com/sun/corba/se/impl/io/ObjectStreamClass.java':[
282,
],
'project:com/sun/corba/se/impl/javax/rmi/CORBA/Util.java':[
549,
],
'project:com/sun/corba/se/impl/legacy/connection/SocketFactoryContactInfoListIteratorImpl.java':[
103,
],
'project:com/sun/corba/se/impl/oa/poa/POAPolicyMediatorImpl_R_UDS.java':[
110,
],
'project:com/sun/corba/se/impl/oa/poa/POAPolicyMediatorImpl_R_USM.java':[
197,
],
'project:com/sun/corba/se/impl/orb/DataCollectorBase.java':[
185,
],
'project:com/sun/corba/se/impl/orb/ORBImpl.java':[
456,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClass_1_3_1.java':[
265,
],
'project:com/sun/corba/se/impl/presentation/rmi/DynamicMethodMarshallerImpl.java':[
88,
],
'project:com/sun/corba/se/impl/presentation/rmi/IDLTypesUtil.java':[
241,
248,
256,
],
'project:com/sun/corba/se/impl/protocol/CorbaClientRequestDispatcherImpl.java':[
259,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/MessageBase.java':[
213,
751,
],
'project:com/sun/corba/se/impl/transport/CorbaTransportManagerImpl.java':[
218,
],
'project:com/sun/corba/se/impl/transport/SelectorImpl.java':[
296,
],
'project:com/sun/corba/se/spi/ior/iiop/GIOPVersion.java':[
111,
],
'project:com/sun/imageio/plugins/common/LZWStringTable.java':[
171,
],
'project:com/sun/imageio/plugins/jpeg/JPEGImageReader.java':[
482,
507,
815,
],
'project:com/sun/imageio/plugins/jpeg/JPEGImageWriter.java':[
666,
841,
1299,
1310,
],
'project:com/sun/imageio/plugins/jpeg/JPEGMetadata.java':[
426,
437,
469,
1660,
1889,
1941,
2267,
2352,
],
'project:com/sun/imageio/plugins/jpeg/SOSMarkerSegment.java':[
61,
],
'project:com/sun/imageio/plugins/png/PNGMetadata.java':[
1729,
],
'project:com/sun/java/swing/plaf/gtk/GTKLookAndFeel.java':[
1583,
],
'project:com/sun/java/swing/plaf/gtk/GTKStyleFactory.java':[
89,
],
'project:com/sun/java/swing/plaf/motif/MotifComboBoxUI.java':[
352,
],
'project:com/sun/java/swing/plaf/windows/WindowsDesktopManager.java':[
76,
],
'project:com/sun/java/swing/plaf/windows/WindowsFileChooserUI.java':[
841,
],
'project:com/sun/java/swing/plaf/windows/WindowsMenuUI.java':[
98,
],
'project:com/sun/java/swing/plaf/windows/WindowsScrollBarUI.java':[
176,
],
'project:com/sun/jmx/interceptor/DefaultMBeanServerInterceptor.java':[
1741,
],
'project:com/sun/jmx/mbeanserver/JmxMBeanServer.java':[
1441,
],
'project:com/sun/jmx/mbeanserver/MBeanAnalyzer.java':[
259,
],
'project:com/sun/jmx/mbeanserver/MXBeanSupport.java':[
169,
],
'project:com/sun/jmx/remote/internal/ServerNotifForwarder.java':[
198,
],
'project:com/sun/jmx/snmp/IPAcl/AclImpl.java':[
229,
256,
],
'project:com/sun/jmx/snmp/SnmpParameters.java':[
152,
],
'project:com/sun/jmx/snmp/agent/SnmpMib.java':[
256,
],
'project:com/sun/jmx/snmp/agent/SnmpRequestTree.java':[
212,
],
'project:com/sun/jmx/snmp/daemon/CommunicatorServer.java':[
1055,
1086,
1099,
],
'project:com/sun/jmx/snmp/daemon/SnmpInformRequest.java':[
617,
],
'project:com/sun/jmx/snmp/daemon/SnmpQManager.java':[
185,
280,
],
'project:com/sun/jmx/snmp/internal/SnmpEngineImpl.java':[
418,
],
'project:com/sun/org/apache/bcel/internal/classfile/Utility.java':[
481,
],
'project:com/sun/org/apache/bcel/internal/generic/ReferenceType.java':[
131,
143,
164,
],
'project:com/sun/org/apache/regexp/internal/RE.java':[
1454,
],
'project:com/sun/org/apache/regexp/internal/REProgram.java':[
126,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/DecimalFormatting.java':[
66,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/FunctionCall.java':[
728,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/LiteralElement.java':[
540,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/LocationPathPattern.java':[
87,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Output.java':[
306,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/ParentLocationPath.java':[
160,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Parser.java':[
969,
1313,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/ProcessingInstruction.java':[
58,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/RelationalExpr.java':[
118,
125,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Step.java':[
309,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Stylesheet.java':[
349,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/XSLTC.java':[
404,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/XslAttribute.java':[
98,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/MethodGenerator.java':[
313,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/BitArray.java':[
107,
162,
189,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/DOMAdapter.java':[
110,
120,
131,
140,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/DocumentCache.java':[
204,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerImpl.java':[
969,
1037,
],
'project:com/sun/org/apache/xerces/internal/dom/AttrNSImpl.java':[
331,
],
'project:com/sun/org/apache/xerces/internal/dom/CoreDocumentImpl.java':[
264,
392,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMNormalizer.java':[
303,
1994,
],
'project:com/sun/org/apache/xerces/internal/dom/DocumentImpl.java':[
927,
1006,
1017,
1055,
1201,
],
'project:com/sun/org/apache/xerces/internal/dom/DocumentTypeImpl.java':[
261,
267,
273,
],
'project:com/sun/org/apache/xerces/internal/dom/ElementNSImpl.java':[
312,
],
'project:com/sun/org/apache/xerces/internal/dom/NodeImpl.java':[
1614,
1632,
],
'project:com/sun/org/apache/xerces/internal/dom/NodeIteratorImpl.java':[
271,
],
'project:com/sun/org/apache/xerces/internal/dom/RangeImpl.java':[
869,
876,
941,
947,
],
'project:com/sun/org/apache/xerces/internal/impl/XML11DocumentScannerImpl.java':[
252,
],
'project:com/sun/org/apache/xerces/internal/impl/XML11NSDocumentScannerImpl.java':[
184,
230,
406,
450,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDTDScannerImpl.java':[
1211,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentFragmentScannerImpl.java':[
834,
870,
1250,
1766,
1975,
2691,
2900,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentScannerImpl.java':[
461,
539,
965,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityScanner.java':[
1172,
1536,
2132,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLNSDocumentScannerImpl.java':[
205,
225,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLNamespaceBinder.java':[
873,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLScanner.java':[
815,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XML11NSDTDValidator.java':[
257,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDProcessor.java':[
905,
1019,
1045,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDValidator.java':[
703,
801,
812,
1129,
1178,
1216,
1718,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLNSDTDValidator.java':[
265,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/models/DFAContentModel.java':[
369,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/models/MixedContentModel.java':[
195,
238,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/models/SimpleContentModel.java':[
220,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/XSSimpleTypeDecl.java':[
922,
923,
979,
1026,
1051,
1052,
1103,
1133,
1139,
1146,
1165,
1220,
1227,
1371,
1383,
1384,
1651,
1659,
1667,
2635,
2642,
2649,
2787,
2820,
],
'project:com/sun/org/apache/xerces/internal/impl/validation/ValidationState.java':[
94,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/XPath.java':[
1954,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/CaseInsensitiveMap.java':[
129,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/RegularExpression.java':[
797,
988,
1661,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaValidator.java':[
1650,
1700,
1849,
1987,
2342,
2384,
2703,
2832,
2847,
2933,
3008,
3029,
3078,
3089,
3190,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSAttributeGroupDecl.java':[
78,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSComplexTypeDecl.java':[
286,
294,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSConstraints.java':[
438,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSWildcardDecl.java':[
139,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/identity/XPathMatcher.java':[
285,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/CMBuilder.java':[
376,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/XSDFACM.java':[
308,
403,
429,
649,
1108,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/opti/SchemaParsingConfig.java':[
957,
974,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDAbstractIDConstraintTraverser.java':[
154,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDAbstractTraverser.java':[
732,
791,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDAttributeTraverser.java':[
199,
418,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDComplexTypeTraverser.java':[
215,
989,
1086,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDElementTraverser.java':[
510,
518,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDHandler.java':[
593,
717,
722,
1645,
1655,
1755,
3111,
3345,
3609,
3742,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDKeyrefTraverser.java':[
113,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDSimpleTypeTraverser.java':[
129,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDUniqueOrKeyTraverser.java':[
91,
],
'project:com/sun/org/apache/xerces/internal/jaxp/DocumentBuilderImpl.java':[
247,
],
'project:com/sun/org/apache/xerces/internal/jaxp/datatype/XMLGregorianCalendarImpl.java':[
1231,
1249,
1268,
1323,
1337,
1344,
2689,
2697,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/DOMResultAugmentor.java':[
140,
167,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/XMLSchemaValidatorComponentManager.java':[
495,
516,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractDOMParser.java':[
1326,
1486,
1496,
1843,
2463,
2628,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractSAXParser.java':[
427,
857,
],
'project:com/sun/org/apache/xerces/internal/parsers/XML11Configuration.java':[
1494,
1511,
],
'project:com/sun/org/apache/xerces/internal/parsers/XML11DTDConfiguration.java':[
1283,
1300,
],
'project:com/sun/org/apache/xerces/internal/parsers/XML11NonValidatingConfiguration.java':[
1182,
1199,
],
'project:com/sun/org/apache/xerces/internal/util/DOMUtil.java':[
375,
394,
413,
],
'project:com/sun/org/apache/xerces/internal/util/NamespaceSupport.java':[
259,
342,
],
'project:com/sun/org/apache/xerces/internal/util/URI.java':[
925,
],
'project:com/sun/org/apache/xerces/internal/xinclude/MultipleScopeNamespaceSupport.java':[
136,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
1002,
1677,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBaseIterators.java':[
408,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDocumentImpl.java':[
1237,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/dom2dtm/DOM2DTM.java':[
259,
383,
403,
639,
711,
],
'project:com/sun/org/apache/xml/internal/resolver/Catalog.java':[
1246,
1359,
1445,
1604,
1743,
1769,
1901,
1927,
],
'project:com/sun/org/apache/xml/internal/resolver/readers/OASISXMLCatalogReader.java':[
475,
],
'project:com/sun/org/apache/xml/internal/security/c14n/helper/C14nHelper.java':[
112,
],
'project:com/sun/org/apache/xml/internal/security/c14n/implementations/Canonicalizer20010315Excl.java':[
169,
288,
],
'project:com/sun/org/apache/xml/internal/security/c14n/implementations/NameSpaceSymbTable.java':[
247,
334,
],
'project:com/sun/org/apache/xml/internal/security/keys/storage/implementations/CertsInFilesystemDirectoryResolver.java':[
142,
],
'project:com/sun/org/apache/xml/internal/security/signature/Reference.java':[
583,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/TransformXPath2Filter.java':[
250,
251,
],
'project:com/sun/org/apache/xml/internal/security/utils/CachedXPathFuncHereAPI.java':[
356,
],
'project:com/sun/org/apache/xml/internal/security/utils/HelperNodeList.java':[
85,
],
'project:com/sun/org/apache/xml/internal/security/utils/RFC2253Parser.java':[
487,
],
'project:com/sun/org/apache/xml/internal/security/utils/resolver/ResourceResolverSpi.java':[
170,
],
'project:com/sun/org/apache/xml/internal/security/utils/resolver/implementations/ResolverDirectHTTP.java':[
183,
],
'project:com/sun/org/apache/xml/internal/serialize/BaseMarkupSerializer.java':[
1288,
],
'project:com/sun/org/apache/xml/internal/serialize/DOMSerializerImpl.java':[
1098,
1099,
],
'project:com/sun/org/apache/xml/internal/serialize/TextSerializer.java':[
123,
264,
],
'project:com/sun/org/apache/xml/internal/serialize/XMLSerializer.java':[
969,
],
'project:com/sun/org/apache/xml/internal/serializer/EncodingInfo.java':[
343,
],
'project:com/sun/org/apache/xml/internal/serializer/SerializerBase.java':[
894,
],
'project:com/sun/org/apache/xml/internal/serializer/ToHTMLSAXHandler.java':[
537,
],
'project:com/sun/org/apache/xml/internal/serializer/ToHTMLStream.java':[
997,
],
'project:com/sun/org/apache/xml/internal/serializer/ToStream.java':[
519,
1279,
1905,
],
'project:com/sun/org/apache/xml/internal/serializer/ToUnknownStream.java':[
432,
],
'project:com/sun/org/apache/xml/internal/serializer/utils/URI.java':[
550,
679,
765,
],
'project:com/sun/org/apache/xml/internal/utils/DOMBuilder.java':[
214,
],
'project:com/sun/org/apache/xml/internal/utils/QName.java':[
118,
215,
306,
395,
473,
],
'project:com/sun/org/apache/xml/internal/utils/URI.java':[
420,
570,
699,
785,
],
'project:com/sun/org/apache/xpath/internal/axes/UnionPathIterator.java':[
542,
],
'project:com/sun/org/apache/xpath/internal/axes/WalkerFactory.java':[
638,
],
'project:com/sun/org/apache/xpath/internal/axes/WalkingIterator.java':[
307,
],
'project:com/sun/org/apache/xpath/internal/axes/WalkingIteratorSorted.java':[
122,
],
'project:com/sun/org/apache/xpath/internal/compiler/Lexer.java':[
290,
319,
],
'project:com/sun/org/apache/xpath/internal/compiler/XPathParser.java':[
1599,
1857,
],
'project:com/sun/org/apache/xpath/internal/operations/Variable.java':[
392,
],
'project:com/sun/org/apache/xpath/internal/patterns/ContextMatchStepPattern.java':[
156,
],
'project:com/sun/org/apache/xpath/internal/patterns/StepPattern.java':[
407,
],
'project:com/sun/security/auth/SubjectCodeSource.java':[
298,
350,
],
'project:com/sun/security/auth/module/Krb5LoginModule.java':[
632,
647,
697,
1053,
],
'project:java/awt/AttributeValue.java':[
41,
],
'project:java/awt/Component.java':[
1177,
3069,
3508,
4010,
4696,
4765,
4907,
5487,
5528,
8031,
9350,
9386,
9629,
9639,
],
'project:java/awt/Container.java':[
507,
603,
1955,
2548,
2557,
2808,
],
'project:java/awt/DefaultKeyboardFocusManager.java':[
749,
],
'project:java/awt/Frame.java':[
544,
],
'project:java/awt/GradientPaintContext.java':[
48,
],
'project:java/awt/KeyboardFocusManager.java':[
2137,
2500,
2693,
2934,
],
'project:java/awt/List.java':[
618,
],
'project:java/awt/MultipleGradientPaintContext.java':[
667,
],
'project:java/awt/Robot.java':[
150,
],
'project:java/awt/Toolkit.java':[
2149,
],
'project:java/awt/Window.java':[
854,
1377,
2956,
],
'project:java/awt/datatransfer/SystemFlavorMap.java':[
357,
],
'project:java/awt/event/MouseEvent.java':[
760,
],
'project:java/awt/font/StyledParagraph.java':[
242,
],
'project:java/awt/font/TextMeasurer.java':[
521,
526,
],
'project:java/awt/geom/Arc2D.java':[
1292,
1300,
],
'project:java/awt/image/BufferedImage.java':[
712,
747,
],
'project:java/awt/image/LookupOp.java':[
188,
],
'project:java/awt/image/MultiPixelPackedSampleModel.java':[
331,
],
'project:java/awt/image/RescaleOp.java':[
396,
],
'project:java/awt/print/PrinterJob.java':[
449,
],
'project:java/beans/IndexedPropertyDescriptor.java':[
274,
],
'project:java/beans/Introspector.java':[
516,
],
'project:java/beans/MetaData.java':[
778,
],
'project:java/beans/PropertyDescriptor.java':[
298,
517,
],
'project:java/lang/Character.java':[
4670,
4741,
4775,
4848,
],
'project:java/lang/Class.java':[
935,
2648,
2694,
],
'project:java/lang/StringCoding.java':[
220,
354,
],
'project:java/lang/Thread.java':[
382,
],
'project:java/lang/reflect/Constructor.java':[
513,
],
'project:java/lang/reflect/Field.java':[
980,
],
'project:java/lang/reflect/Method.java':[
591,
],
'project:java/net/AbstractPlainSocketImpl.java':[
613,
],
'project:java/net/IDN.java':[
287,
],
'project:java/net/Inet4Address.java':[
114,
],
'project:java/net/Inet6Address.java':[
272,
277,
303,
308,
],
'project:java/net/InetAddress.java':[
784,
962,
],
'project:java/net/SocketPermission.java':[
302,
601,
857,
991,
],
'project:java/net/SocksSocketImpl.java':[
402,
710,
],
'project:java/net/URI.java':[
1802,
3324,
],
'project:java/net/URLClassLoader.java':[
530,
],
'project:java/net/URLConnection.java':[
1471,
1478,
1485,
1493,
1501,
1556,
],
'project:java/net/URLEncoder.java':[
248,
],
'project:java/nio/charset/Charset.java':[
307,
],
'project:java/nio/charset/CharsetDecoder.java':[
195,
],
'project:java/nio/charset/CharsetEncoder.java':[
195,
],
'project:java/security/BasicPermission.java':[
393,
],
'project:java/security/CodeSource.java':[
411,
],
'project:java/security/Identity.java':[
270,
371,
],
'project:java/security/ProtectionDomain.java':[
415,
],
'project:java/security/Security.java':[
1099,
],
'project:java/security/cert/X509CRLSelector.java':[
625,
646,
656,
],
'project:java/security/cert/X509CertSelector.java':[
2018,
2029,
2040,
2051,
2484,
2489,
],
'project:java/sql/SQLException.java':[
74,
100,
123,
144,
168,
190,
215,
241,
],
'project:java/text/BreakIterator.java':[
559,
],
'project:java/text/DecimalFormat.java':[
2269,
],
'project:java/text/RuleBasedCollator.java':[
411,
516,
532,
],
'project:java/text/SimpleDateFormat.java':[
1477,
1547,
1841,
1911,
1965,
],
'project:java/util/Date.java':[
483,
],
'project:java/util/GregorianCalendar.java':[
2582,
2965,
],
'project:java/util/JapaneseImperialCalendar.java':[
688,
1867,
],
'project:java/util/PropertyPermission.java':[
507,
],
'project:java/util/SimpleTimeZone.java':[
552,
],
'project:java/util/concurrent/ConcurrentLinkedDeque.java':[
1123,
],
'project:java/util/concurrent/ConcurrentLinkedQueue.java':[
452,
],
'project:java/util/concurrent/ForkJoinPool.java':[
1191,
],
'project:java/util/concurrent/ScheduledThreadPoolExecutor.java':[
377,
],
'project:java/util/logging/LogRecord.java':[
547,
],
'project:java/util/regex/Pattern.java':[
3220,
3235,
3249,
3452,
],
'project:javax/imageio/IIOParam.java':[
322,
],
'project:javax/management/MBeanPermission.java':[
1121,
],
'project:javax/management/ObjectName.java':[
956,
],
'project:javax/management/StandardMBean.java':[
1231,
],
'project:javax/management/loading/MLet.java':[
528,
],
'project:javax/management/modelmbean/DescriptorSupport.java':[
931,
],
'project:javax/management/modelmbean/ModelMBeanOperationInfo.java':[
501,
],
'project:javax/management/modelmbean/RequiredModelMBean.java':[
499,
2206,
2216,
2599,
2607,
],
'project:javax/management/monitor/CounterMonitor.java':[
772,
773,
895,
],
'project:javax/management/monitor/GaugeMonitor.java':[
605,
],
'project:javax/management/monitor/Monitor.java':[
1340,
1358,
],
'project:javax/management/monitor/StringMonitor.java':[
442,
],
'project:javax/management/openmbean/OpenMBeanAttributeInfoSupport.java':[
520,
526,
557,
],
'project:javax/management/relation/MBeanServerNotificationFilter.java':[
215,
223,
270,
278,
],
'project:javax/management/relation/RelationNotification.java':[
510,
],
'project:javax/management/relation/RelationService.java':[
2848,
],
'project:javax/management/remote/JMXConnectorFactory.java':[
465,
],
'project:javax/management/remote/JMXConnectorServerFactory.java':[
229,
],
'project:javax/management/remote/JMXServiceURL.java':[
362,
],
'project:javax/management/remote/rmi/RMIConnector.java':[
466,
],
'project:javax/management/remote/rmi/RMIConnectorServer.java':[
526,
],
'project:javax/management/remote/rmi/RMIServerImpl.java':[
223,
],
'project:javax/print/ServiceUI.java':[
252,
],
'project:javax/script/ScriptEngineManager.java':[
427,
],
'project:javax/security/auth/PrivateCredentialPermission.java':[
541,
],
'project:javax/security/sasl/Sasl.java':[
588,
],
'project:javax/sound/midi/MidiSystem.java':[
1459,
],
'project:javax/sound/midi/ShortMessage.java':[
324,
],
'project:javax/sound/sampled/AudioInputStream.java':[
336,
],
'project:javax/sound/sampled/DataLine.java':[
443,
449,
460,
],
'project:javax/sql/rowset/spi/SyncFactory.java':[
774,
],
'project:javax/swing/AbstractButton.java':[
511,
557,
631,
680,
1317,
],
'project:javax/swing/BufferStrategyPaintManager.java':[
380,
610,
],
'project:javax/swing/DefaultDesktopManager.java':[
204,
547,
],
'project:javax/swing/DefaultListSelectionModel.java':[
757,
],
'project:javax/swing/JComponent.java':[
732,
5260,
],
'project:javax/swing/JDesktopPane.java':[
488,
536,
],
'project:javax/swing/JFileChooser.java':[
569,
],
'project:javax/swing/JMenu.java':[
1481,
1582,
],
'project:javax/swing/JPopupMenu.java':[
933,
],
'project:javax/swing/JRootPane.java':[
824,
],
'project:javax/swing/JScrollPane.java':[
1524,
],
'project:javax/swing/JTable.java':[
5287,
5969,
6227,
],
'project:javax/swing/JTree.java':[
1378,
4459,
4477,
5184,
5370,
5388,
],
'project:javax/swing/PopupFactory.java':[
319,
],
'project:javax/swing/ProgressMonitor.java':[
486,
],
'project:javax/swing/RepaintManager.java':[
1064,
1201,
],
'project:javax/swing/RowFilter.java':[
413,
],
'project:javax/swing/ScrollPaneLayout.java':[
971,
996,
],
'project:javax/swing/SwingUtilities.java':[
859,
],
'project:javax/swing/ToolTipManager.java':[
265,
],
'project:javax/swing/UIManager.java':[
1426,
],
'project:javax/swing/plaf/basic/BasicComboBoxUI.java':[
375,
421,
],
'project:javax/swing/plaf/basic/BasicDesktopIconUI.java':[
236,
],
'project:javax/swing/plaf/basic/BasicHTML.java':[
184,
],
'project:javax/swing/plaf/basic/BasicInternalFrameUI.java':[
228,
724,
],
'project:javax/swing/plaf/basic/BasicLookAndFeel.java':[
2279,
],
'project:javax/swing/plaf/basic/BasicOptionPaneUI.java':[
1183,
],
'project:javax/swing/plaf/basic/BasicPopupMenuUI.java':[
864,
879,
1126,
],
'project:javax/swing/plaf/basic/BasicProgressBarUI.java':[
1265,
],
'project:javax/swing/plaf/basic/BasicRootPaneUI.java':[
207,
],
'project:javax/swing/plaf/basic/BasicScrollPaneUI.java':[
965,
976,
1164,
],
'project:javax/swing/plaf/basic/BasicSpinnerUI.java':[
1006,
],
'project:javax/swing/plaf/basic/BasicTabbedPaneUI.java':[
274,
2259,
2434,
2502,
2946,
3111,
],
'project:javax/swing/plaf/basic/BasicTableUI.java':[
667,
883,
898,
1290,
],
'project:javax/swing/plaf/basic/BasicTextUI.java':[
603,
2155,
],
'project:javax/swing/plaf/basic/BasicTreeUI.java':[
407,
3382,
3586,
3656,
3989,
],
'project:javax/swing/plaf/metal/MetalFileChooserUI.java':[
772,
],
'project:javax/swing/plaf/metal/MetalLookAndFeel.java':[
2491,
2497,
],
'project:javax/swing/plaf/metal/MetalRootPaneUI.java':[
920,
922,
],
'project:javax/swing/plaf/metal/MetalTabbedPaneUI.java':[
1141,
1184,
],
'project:javax/swing/plaf/metal/MetalTitlePane.java':[
943,
],
'project:javax/swing/plaf/metal/MetalToolBarUI.java':[
336,
],
'project:javax/swing/plaf/nimbus/NimbusDefaults.java':[
1164,
],
'project:javax/swing/plaf/synth/ParsedSynthStyle.java':[
116,
],
'project:javax/swing/plaf/synth/SynthInternalFrameTitlePane.java':[
80,
],
'project:javax/swing/plaf/synth/SynthMenuBarUI.java':[
81,
],
'project:javax/swing/plaf/synth/SynthParser.java':[
339,
],
'project:javax/swing/plaf/synth/SynthPopupMenuUI.java':[
75,
],
'project:javax/swing/plaf/synth/SynthRootPaneUI.java':[
96,
],
'project:javax/swing/plaf/synth/SynthSeparatorUI.java':[
93,
],
'project:javax/swing/plaf/synth/SynthSpinnerUI.java':[
128,
],
'project:javax/swing/plaf/synth/SynthSplitPaneDivider.java':[
54,
],
'project:javax/swing/plaf/synth/SynthTabbedPaneUI.java':[
305,
551,
611,
],
'project:javax/swing/plaf/synth/SynthTableHeaderUI.java':[
83,
],
'project:javax/swing/plaf/synth/SynthTableUI.java':[
689,
],
'project:javax/swing/table/DefaultTableCellRenderer.java':[
122,
],
'project:javax/swing/text/ComponentView.java':[
298,
],
'project:javax/swing/text/CompositeView.java':[
267,
],
'project:javax/swing/text/DefaultCaret.java':[
435,
1637,
],
'project:javax/swing/text/DefaultStyledDocument.java':[
1990,
1996,
],
'project:javax/swing/text/FlowView.java':[
341,
],
'project:javax/swing/text/InternationalFormatter.java':[
755,
],
'project:javax/swing/text/MaskFormatter.java':[
738,
],
'project:javax/swing/text/ParagraphView.java':[
1179,
],
'project:javax/swing/text/PlainDocument.java':[
119,
],
'project:javax/swing/text/PlainView.java':[
572,
],
'project:javax/swing/text/View.java':[
700,
741,
782,
1153,
],
'project:javax/swing/text/ZoneView.java':[
314,
],
'project:javax/swing/text/html/AccessibleHTML.java':[
747,
],
'project:javax/swing/text/html/CSS.java':[
2196,
],
'project:javax/swing/text/html/FrameSetView.java':[
77,
],
'project:javax/swing/text/html/HTMLDocument.java':[
1453,
3101,
3139,
3750,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
1321,
2235,
],
'project:javax/swing/text/html/StyleSheet.java':[
3260,
],
'project:javax/swing/text/html/TableView.java':[
574,
],
'project:javax/swing/text/html/parser/ContentModelState.java':[
118,
],
'project:javax/swing/text/html/parser/Parser.java':[
330,
586,
1766,
1776,
1884,
],
'project:javax/swing/text/rtf/RTFGenerator.java':[
230,
571,
584,
],
'project:javax/swing/text/rtf/RTFReader.java':[
263,
373,
488,
],
'project:javax/swing/tree/DefaultTreeCellEditor.java':[
257,
],
'project:javax/swing/tree/DefaultTreeSelectionModel.java':[
240,
456,
],
'project:javax/swing/tree/FixedHeightLayoutCache.java':[
209,
],
}
